spec/index.bs

```diff
@@ -1043,6 +1043,12 @@ a better limit is not specified.
     <tr class=row-continuation><td colspan=4>
         The maximum {{GPUBufferBinding}}.{{GPUBufferBinding/size}} for bindings
         of type {{GPUBindingType/uniform-buffer}}.
+
+    <tr><td><dfn>maxStorageBufferBindingSize</dfn>
+        <td>{{GPUSize32}} <td>Higher <td> 128 Mb
```

Comment:
Should make the unit clear:
```html
        <td>{{GPUSize32}} <td>Higher <td> 128 MiB
```

And probably should show the actual number because that's what the user will request/receive:

```suggestion
        <td>{{GPUSize32}} <td>Higher <td> 134217728 (128 MiB)
```